### DISCUSSION TITLE
Mesh Conversion Issue

### DISCUSSION THREAD

**Discussion by FURUNZ** [SeisSol/SeisSol#1195] [Q&A] [answered]
When using gmsh2gambit as in the Seissol case, I encountered a "bash: can't find command" error. I have installed Gmsh version 4.0 but haven't successfully installed gmsh2gambit. However, I can directly use pumgen to convert the .msh file into PUML format. Will doing so impact the simulation results?

**Comment by davschneller** [comment, marked as answer]
In effect, there shouldn't be any difference—`gmsh2gambit` is de facto obsolete by now, due to the direct route via PUMgen.

(on the contrary, the Gambit format was reported to break large meshes in some other cases; cf. e.g. the issues here https://github.com/SeisSol/PUMGen/issues/77 )

I.e. it should have no effect on the simulation results.
